src/routes/blog/post/netlify-vs-vercel-vs-amplify-vs-appwrite-sites/+page.markdoc

```diff
@@ -0,0 +1,147 @@
+
+So the real question isn’t just *“where do I host my app?”,* it’s *“what kind of experience (and bill) am I signing up for?”* In this blog, we’ll break down **Vercel, Netlify, AWS Amplify, and Appwrite Sites** to see how each fits into the way developers are shipping web apps today.
+
+# Quick overview
+
+- **Vercel**: Think of Vercel as the **framework-native host**. It’s tightly coupled with Next.js, and its entire model revolves around serving modern frontend frameworks at the edge with minimal friction. Backend logic exists, but always as small serverless functions or external APIs.    
+- **Netlify**: The **Jamstack generalist**. Netlify built its reputation on static-first workflows: push code, build, deploy, done. It doesn’t lean into any one framework, instead, it’s about a consistent, git-driven pipeline that works for React, Vue, Svelte, or plain HTML.
+- **AWS Amplify**: Amplify is built to connect your frontend with AWS services. It gives you hosting, but the main focus is making it easy to add things like authentication (Cognito), databases (DynamoDB), file storage (S3), and serverless functions (Lambda). Hosting is just one part; the bigger goal is tying your app into AWS’s cloud tools. 
+- **Appwrite Sites**: The **open-source all-in-one development platform**. Instead of separating hosting from backend, [Appwrite Sites](/products/sites) is an extension of Appwrite’s backend platform: you host the frontend and connect directly to the built-in [auth](/docs/products/auth), [Databases](/docs/products/databases), and [Functions](/docs/products/functions). It’s less about glueing services together and more about having a **cohesive stack under one roof**.
+    
+| Platform | Best fit for | Approach | Lock-In | Open source |
+| --- | --- | --- | --- | --- |
+| **Vercel** | Next.js and modern frontend apps | Framework-optimized, frontend-first | Medium (proprietary infra) | No |
+| **Netlify** | Static sites & Jamstack projects | Git-based deploys, static-first | Medium (plugins, infra) | No |
+| **AWS Amplify** | Teams already in AWS ecosystem | Hosting + AWS backend services | High (AWS services) | No |
+| **Appwrite Sites** | Full-stack apps, open-source users | Hosting + built-in backend stack | Low (self-host or cloud) | Yes |
+
+# Vercel
+
+## Pricing of Vercel
+
+- **Free**: Hobby projects, personal sites (limited bandwidth & function executions).
+- **Pro**: ~$20/user/month, adds better bandwidth, analytics, and team features.
+- **Enterprise**: Custom pricing, advanced security, scaling, and SLAs.
+
+## Key features of Vercel
+
+- Optimized for **Next.js** (ISR, edge functions, fast refresh).
+- Global edge network for low-latency delivery.
+- Built-in CI/CD with Git integration.
+- Analytics and observability baked in.
+
+## Pros and cons of Vercel
+
+| Pros | Cons |
+| --- | --- |
+| Best-in-class support for Next.js | Expensive as usage scales |
```

Comment:
I think "Expensive as usage scales" needs to be nuanced. Why is it expensive as usage scales, and what makes their pricing different from Appwrite Sites? Maybe we can touch on that somwhere in the article